### ISSUE TITLE
Auto learning rate finder via PyTorch Lightning not working

### ISSUE THREAD

**Issue by mccrinbc-ss** [mccrinbc-ss/conformal_bayes#1]
PyTorch lightning wants the dataloaders currently implemented in the datamodule python files within the model file. Since this is not the case, we cannot use the built-in functionality of determining what ptl thinks is the optimal initial lr. Either need to find a work around or not care about this feature. 

This isn't really a "bug", but more of an inconvenience
